src/instructlab/configuration.py

```diff
@@ -749,6 +763,10 @@ class Config(BaseModel):
         default_factory=_metadata,
         description="Metadata pertaining to the specifics of the system which the Configuration is meant to be applied to.",
     )
+    models: List[_model_config] = Field(
+        default_factory=lambda: [],
+        description="User-defined custom set of models. This allows people to use their own models for the InstructLab model customization process.",
+    )
```

Comment:
sorry, I not sure about the purpose and may not understand correctly, based on the code, user defined `models`, and then need to specify the model id in the models?
why not just specify the model id?

Reply:
@reidliu41 Not sure if I follow your question

Reply:
@reidliu41 it would look something like
```
chat: ...
generate: ...
models:
  - id: 'ibm/granite-7b-lab'
    path: <local/hf path>
    ctx_size: 4096
  - id: ...
```
and then the long term goal would be to have model references in other sections (like in `chat.model` or `generate.model`) start pointing to these model ids instead of having the path directly  so we can centralize all the model info in one place for a given model